helper-libraries/hashing-orders.md

```diff
@@ -0,0 +1,48 @@
+When an order is submitted to the Swap contract, a signature of the order is provided with it, which gives security to the signer that the sender of the order cannot modify the order's contents.. This signature is constructed by calculating the hash of the order, before signing this hash. Due to the use of sub-objects in an order, there are many different ways of hashing an order - however the smart contract only accepts one. This helper library provides functions to calculate hashes used in various parts of the Airswap Protocol.
```

Comment:
```odify the order's contents..``` double period.

Reply:
Capitalize "AirSwap", though would simply say "...hashes used throughout the protocol."